interface/modules/custom_modules/oe-module-claimrev-connect/templates/eligibility.php

```diff
@@ -66,7 +66,7 @@
                 <div class="row">
                     <div class="col-2">
                     
-                        <form method="post" action="/interface/patient_file/summary/demographics.php">
+                        <form method="post" action="../../patient_file/summary/demographics.php">
```

Comment:
as you did here, relative path is also another way to fix rather than the webroot global :)

Reply:
I think this finally fixed it. alot of trial and error to figure it out.